buddyboss/buddyboss-platform
# Fix activity mentions tab goes to fatal error

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #564 .

### How to test the changes in this Pull Request:

1. WP Backend > BuddyBoss > Settings > Activity > Enable Activity tabs
2. Front end -> Go to Profile > Activity > Mentions tab
3. It should open the page with mentions activities.

### Other information:




### Changelog entry

> Activity - Mentions tab goes to 404 fix
